### PR TITLE
Fix path separator on windows

### PR DESCRIPTION
**Pull request by isuruf**
Fixes https://github.com/matthew-brett/delocate/issues/50

### CODE REVIEW

File: delocate/tools.py [hunk 1]
```diff
@@ -377,6 +377,10 @@ def dir2zip(in_dir, zip_fname):
             # Preserve file permissions, but allow copy
             info = zipfile.ZipInfo(in_fname)
             info.filename = relpath(in_fname, in_dir)
+            if os.name == 'nt':
+                # Make the path unix friendly on windows.
+                # PyPI won't accept wheels with windows path separators
+                info.filename = relpath(in_fname, in_dir).replace("\\", "/")
```
Comment (matthew-brett) [inline delocate/tools.py]: Just checking - but this should never have a drive letter in - right?

Comment (isuruf) [reply delocate/tools.py]: Right. It's a relative path, so no drive letters.

File: delocate/tools.py [hunk 2]
```diff
@@ -377,6 +377,10 @@ def dir2zip(in_dir, zip_fname):
             # Preserve file permissions, but allow copy
             info = zipfile.ZipInfo(in_fname)
             info.filename = relpath(in_fname, in_dir)
+            if os.name == 'nt':
```
Comment (matthew-brett) [inline delocate/tools.py]: Would `os.path.sep == '\'` give the same answer?  Maybe it's more obvious?

Comment (isuruf) [reply delocate/tools.py]: You mean `relpath(in_fname, in_dir).replace(os.path.sep, '/')` for all platforms?

Comment (matthew-brett) [reply delocate/tools.py]: No, I meant:

```
if os.path.sep == '\':
```

instead of:

```
if os.name == 'nt':
```

**Comment by matthew-brett** [comment]
Great - thanks - let's give it a shot.

**Pull request merged**